Makefile

```diff
@@ -1,61 +1,176 @@
+  $(UUID)/convenience.js \
+  $(UUID)/compat.js \
+  $(UUID)/gpu_usage.sh
+
+# ---------
+# variables
+# ---------
+
 ifeq ($(strip $(DESTDIR)),)
-	INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+  INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+  SUDO=
 else
-	INSTALLBASE = $(DESTDIR)/usr/share/gnome-shell/extensions
+  INSTALLBASE = $(DESTDIR)usr/share/gnome-shell/extensions
+  SUDO=sudo
 endif
-INSTALLNAME = system-monitor@paradoxxx.zero.gmail.com
 
-# The command line passed variable VERSION is used to set the version string
-# in the metadata and in the generated zip-file. If no VERSION is passed, the
-# current commit SHA1 is used as version number in the metadata while the
-# generated zip file has no string attached.
 ifdef VERSION
-	VSTRING = _v$(VERSION)
+  VSTRING = _v$(VERSION)
 else
-	VERSION = $(shell git rev-parse HEAD)
-	VSTRING =
+  VERSION = $(shell git rev-parse HEAD)
+  VSTRING =
 endif
 
-all: extension
+# VERBOSE level
+
+ifeq ($(V),1)
+  Q =
+  VV = -v
+else
+  Q = @
+  VV =
+endif
+
+# -------
+# macros
+# -------
+
+# usage: $(call reload-extension $(UUID))
+
+reload-extension = $(shell gnome-shell-extension-tool -r $(1))
+
+# usage: $(call msg,INFO,'lorem ipsum')
+msg = @printf '  [%-12s] %s\n' '$(1)' '$(2)'
+
+
+# -------
+# targets
+# -------
+
+# is there anymore use of the (old) 'all' target?
+# PHONY += all
+# all: extension
+
+PHONY += help
+help:
+	@echo  'Install or remove (and reload) of the extension, for the local user'
+	@echo  'or as admin for all users:'
+	@echo  ''
+	@echo  '  make [install|remove]                        # for the local user'
+	@echo  '  make DESTDIR=/ [install|remove] clean        # as admin for all users'
+	@echo  ''
+	@echo  'Use environment VERSION=n.m to set verison string in the metadata and in'
+	@echo  'the generated zip-file explicit.  If no VERSION is passed, the current'
+	@echo  'commit SHA1 is used as version number in the metadata while the generated'
+	@echo  'zip file has no string attached.'
+	@echo  ''
+	@echo  'Other targets are:'
+	@echo  ''
+	@echo  '  zip-file  - build and zip ./$(UUID).zip'
+	@echo  '  reload    - reload extension $(UUID)'
+	@echo  '  clean     - remove most generated files'
+	@echo  '  extension - remove most generated files'
+	@echo  '  translate - generate translation from po/ files'
+	@echo  ''
+	@echo  'control verbosity:'
+	@echo  ''
+	@echo  '  make V=0 [targets] -> quiet build (default)'
+	@echo  '  make V=1 [targets] -> verbose build'
+
+
+PHONY += install remove
+
+install: remove build
+	$(call msg,$@,$(SUDO) $(INSTALLBASE)/$(INSTALLNAME))
+	$(Q) $(SUDO) mkdir -p $(INSTALLBASE)/$(INSTALLNAME)
+	$(Q) $(SUDO) cp $(VV) -r ./_build/* $(INSTALLBASE)/$(INSTALLNAME)/
+	$(Q) $(MAKE) reload -s
+	$(call msg,$@,OK)
 
-clean:
-	rm -f ./$(UUID)/schemas/gschemas.compiled
+remove:
+	$(call msg,$@,$(SUDO) $(INSTALLBASE)/$(INSTALLNAME))
+	$(Q) $(SUDO) rm $(VV) -fr $(INSTALLBASE)/$(INSTALLNAME)
+	$(Q) $(MAKE) -s reload
```

Comment:
This line puts the `-s` before the reload and line 100 puts it after.  These should be consistent.

Reply:
Another good suggestion / fixed in a91c9ca